packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -343,6 +343,12 @@ abstract class BindingBase {
     );
   }
 
+  /// Sends an event when a service extension's state is changed. Clients should
```

Comment:
nit: doc comment style is that first paragraph should just be one sentence.

Reply:
Done.